docs/management/connectors/action-types/opsgenie.asciidoc

```diff
@@ -0,0 +1,173 @@
+[role="xpack"]
+[[opsgenie-action-type]]
+=== Opsgenie connector and action
+++++
+<titleabbrev>Opsgenie</titleabbrev>
+++++
+
+The Opsgenie connector uses the https://docs.opsgenie.com/docs/alert-api[Opsgenie Alert API].
+
+[float]
+[[opsgenie-connector-configuration]]
+==== Connector configuration
+
+Opsgenie connectors have the following configuration properties.
+
+Name::      The name of the connector. The name is used to identify a  connector in the management UI connector listing, or in the connector list when configuring an action.
+URL::     The Opsgenie URL: https://api.opsgenie.com. For the EU instance of Opsgenie use: https://api.eu.opsgenie.com. If you are using the <<action-settings, `xpack.actions.allowedHosts`>> setting, make sure the hostname is added to the allowed hosts.
```

Comment:
IMO since this URL is not something we control, we could just provide these as example URLs. Something like this:
```suggestion
URL:: The Opsgenie URL. For example, https://api.opsgenie.com or https://api.eu.opsgenie.com. If you are using the <<action-settings,`xpack.actions.allowedHosts`>> setting, make sure the hostname is added to the allowed hosts.
```